cli/src/main/java/io/kestra/cli/commands/flows/namespaces/FlowNamespaceExtractCommand.java

```diff
@@ -0,0 +1,54 @@
+package io.kestra.cli.commands.flows.namespaces;
+
+import io.kestra.cli.AbstractApiCommand;
+import io.kestra.cli.commands.flows.FlowValidateCommand;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.MutableHttpRequest;
+import io.micronaut.http.client.exceptions.HttpClientResponseException;
+import io.micronaut.http.client.netty.DefaultHttpClient;
+import lombok.extern.slf4j.Slf4j;
+import picocli.CommandLine;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+@CommandLine.Command(
+    name = "extract",
+    description = "extract namespace flows",
+    mixinStandardHelpOptions = true
+)
+@Slf4j
+public class FlowNamespaceExtractCommand extends AbstractApiCommand {
+    private static final String DEFAULT_FILE_NAME = "flows.zip";
+
+    @CommandLine.Parameters(index = "0", description = "the namespace of templates to extract")
+    public String namespace;
+
+    @CommandLine.Parameters(index = "1", description = "the directory to extract the file to")
+    public Path directory;
+
+    @Override
+    public Integer call() throws Exception {
+        super.call();
+
+        try(DefaultHttpClient client = client()) {
+            MutableHttpRequest<Object> request = HttpRequest
+                .GET("/api/v1/flows/extract/by_query?namespace=" + namespace).accept(MediaType.APPLICATION_OCTET_STREAM);
```

Comment:
```suggestion
                .GET("/api/v1/flows/extract/query?namespace=" + namespace).accept(MediaType.APPLICATION_OCTET_STREAM);
```

Reply:
Well, see your comment bellow

> Totally agree, /extract/by-ids and /extract/by-query is better I think

I'm OK to change it but as extract is a verb by_query reads better than just query.

Reply:
Moreover, if we keep `by-ids` we should use `by-query`